pkg/operator/controller/status.go

```diff
@@ -52,15 +51,23 @@ func (r *reconciler) syncOperatorStatus() error {
 			Name:     ns.Name,
 		},
 	}
-	co.Status.Versions = []configv1.OperandVersion{
-		{
-			Name:    "operator",
-			Version: version.OperatorVersion,
-		},
-		{
-			Name:    version.RouterOperandName,
-			Version: r.RouterImage,
-		},
+
+	if len(r.OperatorReleaseVersion) > 0 {
+		// An available operator resets release version
+		for _, condition := range co.Status.Conditions {
+			if condition.Type == configv1.OperatorAvailable && condition.Status == configv1.ConditionTrue {
+				co.Status.Versions = []configv1.OperandVersion{
+					{
+						Name:    "operator",
+						Version: r.OperatorReleaseVersion,
+					},
+					{
+						Name:    "router",
```

Comment:
Should we go ahead and start getting in line with https://github.com/openshift/api/pull/219? e.g. router -> ingress-controller?

Reply:
+1 on the comment above from @ironcladlou As a newbie, I have find `router` and `clusteringress` confusing. I think `ingress-controller` and `ingress-controller-operator` will be easier for users with a k8s background to understand.

Reply:
ok, changed operand name to 'ingress-controller'